terraform/Makefile

```diff
@@ -0,0 +1,22 @@
+SHELL=/bin/bash
+REGION?="us-west-2"
+
+prepare:
+	$(eval AWS=$(shell echo pass hashbang/aws/terraform))
```

Comment:
Going to shell just for `echo` is a mite silly.  Also, there should be no need to call `pass` twice.